### DISCUSSION TITLE
css when build starter project

### DISCUSSION THREAD

**Discussion by DJafari** [nuxt/nuxt#17707] [Questions]
i just build my project without any css, but in `.nuxt/dist/client/` i have a .css file  with some styles ( about 7.7kb ), what is this ? how can remove this ?

i guess it's for `NuxtWelcome` component, but i don't use this component in my app, how can remove this ?

**Comment by procrates** [comment]
I belive this is for error page?

Edit: Yep it is corolated to the .mjs file in the same directory. It is internal css to style the outputs of ex. errors. "Go back home" button is one example.
Just search that file for any class name you find in the css. 

Also I dont think its a good idea to delete that file. But if you really want to you can modify the nuxt package found in @nuxt/ui-templates, or remove them, but i dont know what that will do:)

**Reply by DJafari** [reply]
@procrates 
no this isn't good idea, i think framework should be add a option to disable auto import this

**Reply by procrates** [reply]
Alright. But then you would need to make yourself a error handler, or not. I do believe that react, angular and most high end frameworks have error handling built in along with css to style it.
